content/monitors/faq/how-to-update-anomaly-monitor-timezone.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to update an anomaly detection monitor to account for local time zone?
+kind: faq
+disable_toc: true
+further_reading:
+- link: "/monitors/monitor_types/anomaly/"
+  tag: "Documentation"
+  text: "Create an anomaly monitor"
+- link: "monitors/notifications"
+  tag: "Documentation"
+  text: "Configure monitor notifications"
+---
+
+1.  You need the following information to make the update request through the monitor API:
+  - Your [Datadog API key and APP key][1] for authentication
+  - The monitor ID and query from your anomaly detection monitor:
+    {{< img src="monitors/faq/anomaly_monitor_timezone.png" alt="Monitor ID and Query" responsive="true">}}
+  - The TZ identification string for the time zone related to your metric, for example `America/New_York` or `Europe/Paris`. Locate your preferred time zone in the TZ column on the [List of tz database time zones][2] (canonical format recommended).<br><br>
+2. Create an updated version of the monitor query by adding a time zone argument to the anomalies() function call.
```

Comment:
wrapping time zone into `timezone` would make it more clear it's the name of the argument.